src/services/install/default-arguments.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios'
+import { Logger } from '../../logger'
+import fs from 'fs'
+
+const logger = Logger('install-default-arguments')
+
+export async function defaultArguments() {
+
+    const gitHubResponse = (await axios.get(
```

Comment:
in our JS services we prefer using our Request https://github.com/Backendless/Request

```
const gitHubResults = await BackedlessRequest
  .get('https://hub.docker.com/v2/repositories/backendless/bl-server/tags')
  .query({ page_size: 25, page: 1, ...})


```